src/otx/core/model/detection.py

```diff
@@ -61,6 +62,7 @@ def __init__(
             torch_compile=torch_compile,
         )
         self.tile_config = TileConfig()
+        self.confidence_threshold: float | None = confidence_threshold
```

Comment:
Is this already captured by `self.hparams`? Even if not, please store this in `self.hparams` as the same place as `best_confidence_threshold`.

Reply:
Can we rename this to `fixed_confidence_threshold` to make it clear?
```suggestion
        self.fixed_confidence_threshold: float | None = fixed_confidence_threshold
```